Morcoso_Daisy/script.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener("DOMContentLoaded", function () {
+    let count = 0;
+    let timerInterval;
+    let timerDisplay = document.getElementById("timer");
+    let countList = document.getElementById("count_list");
+  
+    function startTimer() {
+      if (!timerInterval) {
+        timerInterval = setInterval(() => {
+          count++;
+          timerDisplay.textContent = count;
+        }, 1000);
+      }
+    }
+  
+    function pauseTimer() {
+      clearInterval(timerInterval);
+      timerInterval = undefined;
+    }
+  
+    function resumeTimer() {
+      if (!timerInterval) {
+        startTimer();
+      }
+    }
+  
+    function stopTimer() {
+      clearInterval(timerInterval);
+      timerInterval = undefined;
+      count = 0;
+      timerDisplay.textContent = count;
+    }
+  
+    function countTimer() {
```

Comment:
I suggest that you can delete this count function in js because it's not needed and it's also not included in the provided instructions.